README.md

```diff
@@ -2,12 +2,12 @@
 
 JavaScript Toolkit for Rich Web Mapping Applications.
 
-[![Build Status](https://travis-ci.org/geoext/geoext3.svg?branch=master)](https://travis-ci.org/geoext/geoext3)
-[![Coverage Status](https://coveralls.io/repos/geoext/geoext3/badge.svg?branch=master&service=github)](https://coveralls.io/github/geoext/geoext3?branch=master)
-[![dependencies Status](https://david-dm.org/geoext/geoext3/status.svg)](https://david-dm.org/geoext/geoext3)
-[![devDependencies Status](https://david-dm.org/geoext/geoext3/dev-status.svg)](https://david-dm.org/geoext/geoext3?type=dev)
+[![Build Status](https://travis-ci.org/geoext/geoext.svg?branch=master)](https://travis-ci.org/geoext/geoext)
+[![Coverage Status](https://coveralls.io/repos/geoext/geoext/badge.svg?branch=master&service=github)](https://coveralls.io/github/geoext/geoext?branch=master)
```

Comment:
I hope coverage gets reported and registered automatically. Otherwise we need to reregister there. Also I do not know if we keep sonme sort of history of past coverages etc.

Reply:
coveralls started new builds here: https://coveralls.io/github/geoext/geoext, but it does not create a valid badge at the moment, maybe there needs to be some sort of reregistering.
It did not keep any history. that can be found under the old address: https://coveralls.io/github/geoext/geoext3

Reply:
thats no problem, I was just curious